go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/autokitteh/starlark-lsp
 
-go 1.19
+go 1.21
```

Comment:
why is this needed?

Reply:
aligned with autokitteh. Do you want me to leave this 1.19?

Reply:
I prefer to state it explicitly. We need go version for goreleaser as well (see Makefile)